app/brave_generated_resources.grd

```diff
@@ -346,7 +346,38 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         Check details
       </message>
 
-      <!-- Adblock contenxt menu -->
+      <!-- Context Menu › Screenshot Tools -->
+      <if expr="use_titlecase">
```

Comment:
just curious but should we be adding this to a `generated_resources` file?

Reply:
By _this_ do you mean strings, or the expression? Both appear to be acceptable from what I can tell (via current file state, and commit history). I do see some efforts to excise subsets of strings (e.g., settings strings) and pace them into their own file (e.g., brave_settings_strings.grdp). That file is then referenced within brave_generated_resources.grd afterwards (so there will still be an edit to this file). I would love to move the strings into their own `grdp` within the feature folder, but there is little precedent for that from what I can tell.

Reply:
Yeah I meant the strings - you could probably add them to `brave/app/screenshots_strings.grdp` but if other people are adding things to this file its probably fine

Reply:
I'm a fan of the distinct-file approach :)